Common/Integration/aws-cdktf-securityhub-integration/README.md

```diff
@@ -0,0 +1,17 @@
+# CDK/Terraform project to deploy Cloud One Integrations - AWS Security Hub Connector  
+
+This is a CDKTF project with a pre-generated Terraform template that can be used to deploy all the required IAM resources in a given AWS account to integrate Cloud One Integrations with AWS Security Hub.
+
+## Requirements
+
+Take note of your Cloud One account id.
```

Comment:
Maybe a note on how to obtain the account id.

Reply:
Good catch! I will fix this...

Reply:
Done!